barretenberg/cpp/src/barretenberg/client_ivc/client_ivc.test.cpp

```diff
@@ -50,7 +50,7 @@ class ClientIVCTests : public ::testing::Test {
      * polynomials will bump size to next power of 2)
      *
      */
-    static Builder create_mock_circuit(ClientIVC& ivc, size_t log2_num_gates = 15)
+    static Builder create_mock_circuit(ClientIVC& ivc, size_t log2_num_gates = 16)
```

Comment:
increased this so that the protogalaxy instances would be the same size. Some have folding verifier/merge recursive verifier so they have different sizes but I guess adding the aggregation object bumps the larger ones over the 2^16 limit.

Reply:
weird because adding 16 more public inputs shouldn't have that much impact theoretically...